lib/couchbase-orm/connection.rb

```diff
@@ -9,8 +8,16 @@ class << self
             attr_accessor :options
         end
 
+        def self.cluster
+            @cluster ||= begin
```

Comment:
couchbase.yml is usuless with that, no ?

Reply:
We changed the configuration since your review @giallon . Now it uses the couchbase.yml file.